### ISSUE TITLE
Add sensors support with count of tasks in a specific list

### ISSUE THREAD

**Issue by black-roland** [black-roland/homeassistant-microsoft-todo#8]
Might be useful, for example, to remind about shopping list items when leaving an office.

Subtasks:

- [x] Investigate Todoist sensor implementation;
- [x] Add sensor support similar to [Todoist](https://www.home-assistant.io/integrations/todoist/);
- [x] Expose tasks as sensor attributes.

**Comment by rexyiy** [comment]
Might there be an option to get the complete list of tasks? At least as JSON?

**Comment by black-roland** [comment]
> Might there be an option to get the complete list of tasks? At least as JSON?

Yes, I'd like to implement this option. I didn't have a chance to look into sensor implementation but I think it will be similar to Todoist with attributes like [this](https://www.home-assistant.io/integrations/todoist/#sensor-attributes). So it would be possible to go through `all_tasks` attribute.

But maybe it makes sense to implement configuration differently. For example, add a custom sensor platform:

```yaml
sensor:
  - platform: microsoft_todo
    friendly_name
    list_name: "Home Assistant"
```

This looks more flexible to me.

**Comment by black-roland** [comment]
Base support was implemented as part of #20. I've also created #22 to extend the current implementation.
